braindecode/models/attn_sleep.py

```diff
@@ -0,0 +1,373 @@
+# Authors: Divyesh Narayanan <divyesh.narayanan@gmail.com>
+#
+# License: BSD (3-clause)
+
+import torch
+from torch import nn
+import torch.nn.functional as F
+import numpy as np
+import math
+import copy
+from copy import deepcopy
+
+
+class AttnSleep(nn.Module):
+    """Sleep Staging Architecture from Eldele et al 2021.
```

Comment:
you add 2 publics models here no? 
to be consistent this could be:

SleepStagerEldele2021

?